website/docs/en-US/drawer.md

```diff
@@ -51,6 +51,34 @@ Callout a temporary drawer, from multiple direction
     }
   };
 </script>
+<!--
+<setup>
+
+  import { defineComponent, ref, getCurrentInstance } from 'vue';
+
+  export default defineComponent({
+    setup() {
+      const { proxy } = getCurrentInstance();
+      const drawer = ref(false);
+      const direction = ref('rtl');
+      const handleClose = (done) => {
+        proxy
+          .$confirm('Are you sure you want to close this?')
```

Comment:
Can we use 
```
import { ElMessageBox } from 'element-plus';
ElMessageBox.confirm()
```

to make it clear?

Reply:
yeah, but I think it need to show how to use the ElMessage in vue instance.So I think it need to use `proxy`.

Reply:
To know how to use Message in vue instance, our user could refer to the doc of Message component.

To us, it is better to use `ElMessageBox.confirm()` directlly.

Reply:
Well, if you think that's better, I'll revise it later

Reply:
Yes, please. Cheers